### DISCUSSION TITLE
Affichage reservation N°

### DISCUSSION THREAD

**Discussion by n57b** [JeromeDevome/GRR#279] [Q&A] [answered]
Salut à tous !

Petite question, serait-il possible de désactiver l'affichage du n° de réservation afin de "gagner" encore une ligne ? 

Merci d'avance :)
![Capture](https://github.com/JeromeDevome/GRR/assets/131746162/5ca14087-dd0c-4823-9f23-90279b8d8384)

**Comment by ynaessens** [comment]
Bonjour,
quelle version de Grr ?
En v3.5.1, vous pouvez choisir les éléments à afficher dans les cellules et dans le popup.
Cordialement,
YN

**Comment by n57b** [comment]
Oh my bad je n'ai pas donné toutes les informations ..

GRR 4.0.3

Concernant les éléments à afficher, j'ai tout désactiver (enfin sauf Bénéficiaire et Horaire) mais impossible de trouver une chose correspondant a ce n° ^^

Cordialement

**Comment by ynaessens** [comment, marked as answer]
Oui, en v4.0.3 il faut entrer dans le code pour modifier cela. Voyez du côté de la fonction affichage_planning_complet ou qqch qui ressemble, dans functions.inc.php.
Cordialement 
YN

**Reply by n57b** [reply]
Parfait je te remercie j'ai fini par trouver :D
